frontends/web/src/routes/account/info/signingconfiguration.tsx

```diff
@@ -37,64 +39,110 @@ interface State {
         }
+        return info.ethereumSimple;
     }
 
     public render(
-        { t,
-          info,
+        { children,
+          account,
+          code,
+          t,
           signingConfigIndex,
         }: RenderableProps<Props>,
-        { canVerifyExtendedPublicKey }: State) {
-        if (info.bitcoinSimple === undefined) {
-            return null;
-        }
+        { canVerifyExtendedPublicKey }: State
+    ) {
+        const config = this.getSimpleInfo();
+        const bitcoinBased = isBitcoinBased(account.coinCode);
         return (
             <div className={style.address}>
-                <div key={info.bitcoinSimple.keyInfo.xpub}>
-                    <h2>{this.scriptTypeTitle(info.bitcoinSimple.scriptType)}</h2>
-                    <label className="labelLarge">{t('accountInfo.extendedPublicKey')}</label>
-                    <QRCode data={info.bitcoinSimple.keyInfo.xpub} />
-                    <div className={style.textareaContainer}>
-                        <CopyableInput value={info.bitcoinSimple.keyInfo.xpub} flexibleHeight />
-                    </div>
-                    <div className="buttons">
-                        {
-                            canVerifyExtendedPublicKey && (
-                                <Button primary onClick={() => this.verifyExtendedPublicKey(signingConfigIndex)}>
-                                    {t('accountInfo.verify')}
-                                </Button>
-                            )
-                        }
+                <div className={style.qrCode}>
+                    { bitcoinBased ? (
+                        <QRCode
+                            data={config.keyInfo.xpub} />
+                    ) : null }
+                </div>
+                <div className={style.details}>
+                    <div className="labelLarge">
+                        { account.isToken ? null : (
+                            <p key="accountname" className={style.entry}>
+                                {/* borrowing translation from accountSummary */}
+                                <strong>{t('accountSummary.name')}:</strong>
+                                <span>{account.name}</span>
+                            </p>
+                        )}
+                        <p key="keypath" className={style.entry}>
+                            <strong>Keypath:</strong>
+                            <code>{config.keyInfo.keypath}</code>
+                        </p>
+                        { ('scriptType' in config) ? (
+                            <p key="accountType" className={style.entry}>
+                                <strong>Type:</strong>
+                                <span>{getScriptName(config.scriptType)}</span>
+                            </p>
+                        ) : null}
+                        { ('scriptType' in config) ? (
+                            <p key="accountType" className={style.entry}>
```

Comment:
duplicate key (i got warnings in the console).

Reply:
Good catch thanks